interface/ceed-preconditioning.c

```diff
@@ -606,13 +614,15 @@ static int CeedSingleOperatorAssemble(CeedOperator op, CeedInt offset, CeedVecto
     }
   }
 
-  // Assemble QFunction
+  // Assemble QFunction (backend version for op fallback parent is faster, if it exists)
+  CeedOperator        op_parent;
   CeedInt             layout_qf[3];
   const CeedScalar   *assembled_qf_array;
   CeedVector          assembled_qf        = NULL;
   CeedElemRestriction assembled_elem_rstr = NULL;
 
-  CeedCall(CeedOperatorLinearAssembleQFunctionBuildOrUpdate(op, &assembled_qf, &assembled_elem_rstr, CEED_REQUEST_IMMEDIATE));
+  CeedCall(CeedOperatorGetFallbackParent(op, &op_parent));
+  CeedCall(CeedOperatorLinearAssembleQFunctionBuildOrUpdate(op_parent, &assembled_qf, &assembled_elem_rstr, CEED_REQUEST_IMMEDIATE));
```

Comment:
I would double check - I don't think we need this here? Isn't getting the parent part of this function call internally?

Reply:
Fixed, reverted